act-rules-format.bs

```diff
@@ -52,66 +48,91 @@ The separation between atomic rules and composite rules creates a division of re
+  <p>If any one of these rules passes, the outcome of the composite rule for the table cell is "passed".</p>
 </div>
 
+The separation between atomic rules and composite rules creates a division of responsibility. Atomic rules test if web content is correctly implemented in a particular solution. Composite rules test if a combination of outcomes from atomic rules is sufficient for an accessibility requirement to be conformant or not.
+
+Not all atomic rules have to be part of a composite rule. Only if the outcome of multiple atomic rules has to be combined in order to determine (non-)conformance of the [Accessibility Requirements](#accessibility-requirements) can composite rules be used. An atomic rule MAY be part of multiple composite rules.
+
 
 ACT Rule Structure {#act-rule-structure}
 ===============================
 
-ACT Rules MUST be written in an format that conforms to the Web Content Accessibility Guidelines [[WCAG]] or a comparable accessibility guidelines recommendation. ACT test cases are allowed to contain inaccessible content. If any test case contains accessibility issues listed in [WCAG 2.1 Section 5.2.5 Non-Interference](https://www.w3.org/TR/WCAG21/#cc5), users MUST be warned of this in advance. Using an accessibility format supports people with disabilities. It also makes internationalization of ACT rules easier.
-
-An ACT Rule MUST consist of the following items.
+An ACT Rule MUST consist of the following items:
 
 * Descriptive Title
 * [Rule Identifier](#rule-identifier)
-* [Rule Description](#structure-description)
+* [Rule Description](#rule-description)
 * [Rule type](#rule-types)
-* [Accessibility Requirements](#structure-accessibility-requirements), if any
+* [Accessibility Requirements](#accessibility-requirements), if any
 * [Aspects Under Test](#input-aspects) (for atomic rules) OR [Atomic Rules List](#atomic-rules-list) (for composite rules)
 * [Applicability](#applicability)
 * [Expectations](#expectations)
-* [Limitations, Assumptions or Exceptions](#structure-limitations-assumptions-exceptions), if any
-* [Accessibility Support](#acc-support) (optional)
+* [Limitations, Assumptions or Exceptions](#limitations-assumptions-exceptions), if any
+* [Accessibility Support](#accessibility-support) (optional)
+* [Evaluation Instructions](#evaluation-instructions)
+* [Test Cases](#test-cases)
+
+ACT Rules MUST be written in an format that conforms to the Web Content Accessibility Guidelines [[WCAG]] or a comperable accessibility guidelines recommendation. ACT Rule [test cases](#test-cases) are allowed to contain inaccessible content. If any test case contains accessibility issues listed in [WCAG 2.1 Section 5.2.5 Non-Interference](https://www.w3.org/TR/WCAG21/#cc5), users MUST be warned of this in advance. Using an accessibility format supports people with disabilities. It also makes internationalisation of ACT rules easier.
```

Comment:
Editorial:

- First sentence: ...written in "a" format, not "an" format   and "comparable" instead of "comperable"
- First sentence: Suggest you say "comparable accessibility standard" since not all are called "guidelines" or are a "recommendation", like the EN 301 549 or other non-W3C standard.
- You should probably provide an explanation of why there MUST be a warning if the (do you really mean test case or do you mean rule?) contains accessibility issues listed in the Non-interference section of WCAG.